src/filters/lcs.js

```diff
@@ -81,11 +81,7 @@ const get = function (array1, array2, match, context) {
     match || defaultMatch,
     innerContext,
   );
-  const result = backtrack(matrix, array1, array2, innerContext);
-  if (typeof array1 === 'string' && typeof array2 === 'string') {
-    result.sequence = result.sequence.join('');
-  }
-  return result;
+  return backtrack(matrix, array1, array2, innerContext);
```

Comment:
`array1` and `array2` are never strings based on the usages of `lcs.get`. `lcs.get` is not exported so there is no concern of it being used differently by users.